### PR TITLE
[Rollup] Remove builders from RollupJobConfig

### PR DESCRIPTION
**Pull request by tlrx**
This pull request removes the builders from the last Rollup configuration object.

### CODE REVIEW

File: x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/RollupJobConfig.java [hunk 1]
```diff
@@ -30,61 +29,112 @@
...
-    RollupJobConfig(String id, String indexPattern, String rollupIndex, String cron, int pageSize, GroupConfig groupConfig,
-                    List<MetricConfig> metricsConfig, TimeValue timeout) {
+    public RollupJobConfig(final String id,
+                           final String indexPattern,
+                           final String rollupIndex,
+                           final String cron,
+                           final int pageSize,
+                           final GroupConfig groupConfig,
+                           final List<MetricConfig> metricsConfig,
+                           final TimeValue timeout) {
```
Comment (jimczi) [inline x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/RollupJobConfig.java]: Maybe add Nullable here ?

**Review by jimczi — APPROVED** [review]
LGTM, thanks @tlrx !

**Comment by tlrx** [comment]
@elasticmachine test this please

File: x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/RollupJobConfig.java (hunk 1 above)
Comment (tlrx) [reply x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/RollupJobConfig.java]: Sure

**Comment by tlrx** [comment]
Thanks @jimczi !

**Pull request merged**